django/core/checks/__init__.py

```diff
@@ -5,6 +5,7 @@
 from .registry import Tags, register, run_checks, tag_exists
 
 # Import these to force registration of checks
+import django.core.checks.asgi  # NOQA isort:skip
```

Comment:
Could we call it `async_safety` rather than `asgi`? I know `async` is a reserved word, but `asgi` is sort of misleading, this happens under synchronous environments too

Reply:
Thanks for the guidance :+1: 

I'll do some research on how other open-source projects handle the Python 3.7 `async`/`await` reserved word breaking change and put it on a discussion. For now, I'll continue on working the documentation and release note